src/tcptunnel.c

```diff
@@ -280,6 +280,7 @@ int wait_for_clients(void)
 	if(settings.client_addr && (strcmp(inet_ntoa(rc.client_addr.sin_addr), options.client_addr) != 0)) {
 		if (settings.log)
 			printf("> %s tcptunnel: refused request request from %s\n", get_current_timestamp(), inet_ntoa(rc.client_addr.sin_addr));
+		close(rc.client.socket);
```

Comment:
I do not believe so, unless the link is being bombarded by scanners, however it is not impossible.